[Bug] generate-summary-report.sh: jq の // 演算子がオブジェクト構築内でシンタックスエラーになる

## 概要

`generate-summary-report.sh` の jq クエリ内で `//`（alternative operator）をオブジェクト構築内で使用した際にシンタックスエラーが発生し、ジョブが失敗する。

## 再現環境

- ワークフロー: `⑤ 統合プロジェクト分析`
- リポジトリ: `lurest-inc/github-projects-starter-kit`
- 実行ログ: https://github.com/lurest-inc/github-projects-starter-kit/actions/runs/23200412054

## エラーメッセージ

```
jq: error: syntax error, unexpected //, expecting '}' (Unix shell quoting issues?) at <top-level>, line 4:
      status: .[0].status // "(未設定)",
jq: 1 compile error
scripts/generate-summary-report.sh: line 156: echo: write error: Broken pipe
##[error]Process completed with exit code 3.
```

## 原因

`scripts/generate-summary-report.sh` 156行目付近のステータス別集計処理で、jq のオブジェクト構築 `{}` 内で `//` 演算子を使用している。一部の jq バージョンではオブジェクト構築内の `//` がパーサーに正しく解釈されず、シンタックスエラーとなる。

```jq
# 問題箇所
status: .[0].status // "(未設定)",
```

## 修正方針

`//` 演算子部分を括弧で明示的にグルーピングする。

```jq
# 修正案
status: (.[0].status // "(未設定)"),
```

## 関連ファイル

- `scripts/generate-summary-report.sh`（156行目付近）

--- a/scripts/generate-summary-report.sh
+++ b/scripts/generate-summary-report.sh
@@ -148,7 +148,7 @@ read -r OPEN_COUNT CLOSED_COUNT MERGED_COUNT < <(echo "${ITEMS}" | jq -r '[
 STATUS_SUMMARY=$(echo "${ITEMS}" | jq --argjson total "${TOTAL_COUNT}" "${JQ_STATUS_ORDER}"'
   sort_by(.status // "(未設定)") | group_by(.status // "(未設定)")
   | map({
-      status: .[0].status // "(未設定)",
+      status: (.[0].status // "(未設定)"),
       count: length,
       percentage: (if $total > 0 then (length / $total * 1000 | round / 10) else 0 end)
     })
@@ -187,7 +187,7 @@ if [[ "${HAS_EFFORT}" == "true" ]]; then
   EFFORT_SUMMARY=$(echo "${ITEMS}" | jq "${JQ_STATUS_ORDER}"'
     sort_by(.status // "(未設定)") | group_by(.status // "(未設定)")
     | map({
-        status: .[0].status // "(未設定)",
+        status: (.[0].status // "(未設定)"),
         estimated_hours: ([.[] | .estimated_hours // 0] | add),
         actual_hours: ([.[] | .actual_hours // 0] | add)
       })

